samples/getting_started/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+// Update to your credentials
```

Comment:
In our QS, we recommend exporting the credentials as a .env or directly to export your credentials using an export command.  Do you think we should really put these credentials inline in the same file as the code itself when we're trying to give a repo that customers might assume follows best practices?